<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminOptionDialog</class>
 <widget class="QDialog" name="AdminOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Option</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color:white;
	border: 0;
}
QLabel[cate=&quot;ipSep&quot;] {
	border: 0;
	background-color:white;
}</string>
     </property>
     <property name="title">
      <string>Server address</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP address of server:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QcIPWidget" name="qcIPWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="adminAccountGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Change admin account on this workstation only</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Login name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="loginNameLineEdit">
        <property name="inputMask">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Old password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="oldPwdLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>New password:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="newPwdLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Confirm password:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="confirmPwdLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="staticMessageLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="changeButton">
       <property name="text">
        <string>Change</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource/clientApp.qrc">
         <normaloff>:/icons/icons/Common/edit.png</normaloff>:/icons/icons/Common/edit.png</iconset>
       </property>
       <property name="cssType" stdset="0">
        <string notr="true">smallBlueButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource/clientApp.qrc">
         <normaloff>:/icons/icons/Common/cancel.png</normaloff>:/icons/icons/Common/cancel.png</iconset>
       </property>
       <property name="cssType" stdset="0">
        <string notr="true">smallBlackButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcIPWidget</class>
   <extends>QWidget</extends>
   <header>qcipwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>adminAccountGroupBox</tabstop>
  <tabstop>loginNameLineEdit</tabstop>
  <tabstop>oldPwdLineEdit</tabstop>
  <tabstop>newPwdLineEdit</tabstop>
  <tabstop>confirmPwdLineEdit</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource/clientApp.qrc"/>
 </resources>
 <connections/>
</ui>
